scb-sdk/src/main/java/io/securecodebox/model/rest/Report.java

```diff
@@ -0,0 +1,202 @@
+/*
+ *
+ *  SecureCodeBox (SCB)
+ *  Copyright 2015-2018 iteratec GmbH
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ * /
+ */
+
+package io.securecodebox.model.rest;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.securecodebox.model.execution.ScanProcessExecution;
+import io.securecodebox.model.execution.Target;
+import io.securecodebox.model.findings.Finding;
+import io.securecodebox.model.findings.Severity;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static java.util.stream.Collectors.counting;
+import static java.util.stream.Collectors.groupingBy;
+
+/**
+ * @author Rüdiger Heins - iteratec GmbH
+ * @since 09.03.18
+ */
+public class Report {
+    @JsonProperty("report_id")
+    @ApiModelProperty(
+            value = "Id for the report.",
+            example = "4e598d7c-5872-4aa0-8e01-770312a00847"
+    )
+    private UUID id = UUID.randomUUID();
+
+    @JsonProperty("security_test_id")
+    @ApiModelProperty(
+            value = "Id for the securityTest.",
+            example = "6f68fe0b-9002-4851-a329-145c489ccbc6"
+    )
+    private UUID securityTestId;
+
+    @JsonProperty("context")
+    @ApiModelProperty(
+            value = "Context references the larger scope the security test. In most cases this is equal to the name of the project, team name or a domain.",
+            example = "Feature Team 1"
+    )
+    private String context;
+
+    @JsonProperty("findings")
+    private List<Finding> findings;
+
+    @JsonProperty("raw_findings")
+    private String rawFindings;
+
+    @JsonProperty("targets")
+    private List<Target> targets;
+
+    @JsonProperty("scanner_type")
+    @ApiModelProperty(
+            value = "The most severe severity in the findings.",
+            example = "nmap"
+    )
+    private String scannerType;
```

Comment:
Thats a good point.
The scannerType is one of the return values of the scanner.
With the current implementation, this will always be the value of the last scanner which was executed.